torch/_inductor/ir.py

```diff
@@ -5761,7 +5761,30 @@ def __init__(
               fp32_output, unary_attr, unary_scalars, unary_algorithm]
         """
         self.has_bias = len(inputs) == 5
-        super().__init__(layout, inputs, constant_args)
+        super().__init__(
+            layout,
+            inputs,
+            constant_args,
+            None,
+            kernel="torch.ops.onednn.qlinear_pointwise",
+            cpp_kernel="onednn::qlinear_pointwise",
+        )
+        self.cpp_kernel_key = "qlinear_pointwise"
+        self.cpp_op_schema = """
```

Comment:
The same comment as in https://github.com/pytorch/pytorch/pull/112373

Reply:
Same as for https://github.com/pytorch/pytorch/pull/112373, let me submit a subsequent PR to clean this up. Created an issue to track it: https://github.com/pytorch/pytorch/issues/112552